### PR TITLE
PLAT-121331: Rename AriaValuetextDecorator.js as AriaValueTextDecorator.js in qa-a11y

### PR DESCRIPTION
**Pull request by ybsung**
- Rename AriaValuetextDecorator.js as AriaValueTextDecorator.js in qa-a11y sample
- Fix JS Doc error in ThemeDecorator

Enact-DCO-1.0-Signed-off-by: YB Sung (yb.sung@lge.com)

### CODE REVIEW

File: samples/qa-a11y/src/views/ArcPicker.js [hunk 1]
```diff
@@ -13,17 +13,15 @@ const ArcPickerView = () => (
 	<>
 		<Section horizontal title="Default">
 			<ArcPicker alt="Normal">{[1, 2, 3, 4, 5]}</ArcPicker>
-			<ArcPicker alt="With slotCenter" slotCenter="SlotCenter">{[1, 2, 3, 4, 5]}</ArcPicker>
+			<ArcPicker alt="With slotCenter" slotCenter="Center">{[1, 2, 3, 4, 5]}</ArcPicker>
```
Comment (MikyungKim) [inline samples/qa-a11y/src/views/ArcPicker.js:16]: Text? instead of Center?

Comment (ybsung) [reply samples/qa-a11y/src/views/ArcPicker.js]: Fixed

File: samples/qa-a11y/src/views/ArcPicker.js [hunk 2]
```diff
@@ -13,17 +13,15 @@ const ArcPickerView = () => (
 	<>
 		<Section horizontal title="Default">
 			<ArcPicker alt="Normal">{[1, 2, 3, 4, 5]}</ArcPicker>
-			<ArcPicker alt="With slotCenter" slotCenter="SlotCenter">{[1, 2, 3, 4, 5]}</ArcPicker>
+			<ArcPicker alt="With slotCenter" slotCenter="Text">{[1, 2, 3, 4, 5]}</ArcPicker>
 			<ArcPicker alt="Disabled" disabled>{[1, 2, 3, 4, 5]}</ArcPicker>
-			<ArcPicker alt="Disabled with slotCenter" disabled slotCenter="SlotCenter">{[1, 2, 3, 4, 5]}</ArcPicker>
```
Comment (MikyungKim) [inline samples/qa-a11y/src/views/ArcPicker.js:18]: Are these safe to remove? No related TCs?

**Comment by MikyungKim** [comment]
When I tap on the item from the side menu, it seems opening other components. So I checked the region and found Marquee height is bigger than the outer layout. Please take a look.
image

**Comment by ybsung** [comment]
I'll fix it in a separate issue.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**